cli/internal/cmd/iamupgradeapply.go

```diff
@@ -91,16 +88,15 @@ func runIAMUpgradeApply(cmd *cobra.Command, _ []string) error {
 	}
 
 	i := iamUpgradeApplyCmd{
-		fileHandler:   fileHandler,
-		configFetcher: configFetcher,
-		log:           log,
```

Comment:
`configFetcher` should stay a variable of `iamUpgradeApplyCmd`, otherwise we cannot stub it for unit tests.